doc/source/reference/c-api/array.rst

```diff
@@ -3469,9 +3473,9 @@ Miscellaneous Macros
 
     Evaluates as True if arrays *a1* and *a2* have the same shape.
 
-.. c:var:: a
+.. c:macro:: a
 
-.. c:var:: b
+.. c:macro:: b
 
 .. c:macro:: PyArray_MAX(a,b)
```

Comment:
Can we remove the `a` and `b` parts now, or does sphinx still need those?

Reply:
We can remove without warnings in sphinx 3.